packages/documentation-v7/src/stories/components/table/table.stories.ts

```diff
@@ -0,0 +1,236 @@
+
+const meta: Meta = {
+  title: 'Components/Table',
+  render: renderTable,
+  parameters: {},
+  args: {
+    variant: [],
+    borderStyle: 'null',
+    caption: 'List of users',
+    captionPlacement: 'bottom',
+    alignment: 'align-top',
+    content: null,
+  },
+  argTypes: {
+    content: {
+      name: 'Content',
+      description: 'Hidden helper arg for dynamically switching the table body',
+      table: {
+        disable: true,
+      },
+    },
+    caption: {
+      name: 'Caption',
+      description: 'A description of the table, very useful for screen reader users.',
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'Caption',
+      },
+    },
+    captionPlacement: {
+      name: 'Caption Placement',
+      description:
+        'Choose where to place the description of the table. If a visual description is not necessary, it can also be hidden visually and still be of value for screen reader users.',
+      control: {
+        type: 'select',
+        labels: {
+          bottom: 'Bottom (default)',
+          top: 'Top',
+          hidden: 'Visually hidden',
+        },
+      },
+      options: ['bottom', 'top', 'hidden'],
+      table: {
+        category: 'Caption',
+      },
+    },
+    borderStyle: {
+      name: 'Border style',
+      description: 'Border style',
+      control: {
+        type: 'radio',
+        labels: {
+          'null': 'Default (underlined)',
+          'table-bordered': 'Bordered',
+          'table-borderless': 'Borderless',
+        },
+      },
+      options: ['null', 'table-bordered', 'table-borderless'],
+      table: {
+        category: 'General',
+      },
+    },
+    variant: {
+      name: 'Variants',
+      description: 'Stylistic table variants.',
+      control: {
+        type: 'check',
+        labels: {
+          'table-striped': 'Striped',
+          'table-hover': 'Hover enabled',
+        },
+      },
+      options: ['table-striped', 'table-hover'],
+      table: {
+        category: 'General',
+      },
+    },
+    alignment: {
+      name: 'Alignment',
+      description:
+        'Vertical alignment options for cell content. Alignment classes can also be set for individual rows or cells.',
+      control: {
+        type: 'select',
+        labels: {
+          'align-top': 'Top (default)',
+          'align-middle': 'Middle',
+          'align-bottom': 'Bottom',
+        },
+      },
+      options: ['align-top', 'align-middle', 'align-bottom'],
+      table: {
+        category: 'General',
+      },
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+function renderTable(args: Args) {
+  const cationTop = args.captionPlacement === 'top' ? 'caption-top' : 'null';
+  return html`
+    <table
+      class=${mapClasses({
+        table: true,
+        [args.borderStyle]: args.borderStyle && args.borderStyle !== 'null',
+        [args.variant]: args.variant && args.variant !== 'null',
```

Comment:
This line is not working as expected.
See classes `table-striped` and `table-hover`:
![image](https://github.com/swisspost/design-system/assets/9716662/58c32877-6648-475f-9455-30859bad6acd)

The value of `args.variant` can ether be '', 'table-striped', 'table-hover' or 'table-striped,table-hover'.
Try changing the logic to someting like this:
```js
const variants = args.variant.split(',').join(' ');

[variants]: variants !== '',
```